Lib/test/test_syntax.py

```diff
@@ -894,19 +894,19 @@
 
 >>> {1, 2, 3} = 42
 Traceback (most recent call last):
-SyntaxError: cannot assign to set display
+SyntaxError: cannot assign to set display. Maybe you meant '==' instead of '='?
 
 >>> {1: 2, 3: 4} = 42
 Traceback (most recent call last):
-SyntaxError: cannot assign to dict display
+SyntaxError: cannot assign to dict display. Maybe you meant '==' instead of '='?
 
 >>> f'{x}' = 42
 Traceback (most recent call last):
-SyntaxError: cannot assign to f-string expression
+SyntaxError: cannot assign to f-string expression. Maybe you meant '==' instead of '='?
```

Comment:
Really "expression" is redundant here -- should be "Cannot assign to f-string."

Reply:
Will fix this in a separate PR, as that error message comes from somewhere else